### DISCUSSION TITLE
Readability could be improved by better use of screen real estate on wide screens

### DISCUSSION THREAD

**Discussion by jcollum-nutrien** [forem/forem#17397] [Q&A]
https://dev.to/dvddpl/dynamodb-dynamic-method-to-insert-or-edit-an-item-5fnh
 
1st screenshot: can’t read a very wide line of text
 
2nd screenshot: still can’t read a very wide line of text even tho I made the window twice as wide

![image](https://user-images.githubusercontent.com/99687439/164566472-f9e28e87-f5be-4528-92d5-67354277e07d.png)

![image](https://user-images.githubusercontent.com/99687439/164566484-2d2aedfa-c48c-4818-9488-ae8af9390e2f.png)

**Comment by amywtlin** [comment]
Hi @jcollum-nutrien - I played around and it looks like the code snipped can be scrolled horizontally. Does that solve the readability issue you brought up, or do I misunderstand your request?

Here's a loom recording for it: https://www.loom.com/share/79e938c7f7374dd1b4a1e4f6aa810f88

Edit: moving your request to Q&A for now, but I can always modify that if I misunderstand your request.
